### PR TITLE
Add routing and test pages

### PR DESCRIPTION
**Pull request by konstrike**
I added routing for navigation, some test pages and bootstrap for top navbar.

### CODE REVIEW

**Review by teodorvoicencu — COMMENTED** [review]
Please provide a description of what you did here. Also install react bootstrap into the project and use it for the top navbar.

**Comment by konstrike** [comment]
I added routing for navigation, some test pages and bootstrap for top navbar.

File: frontend/public/index.html [hunk 1]
```diff
@@ -3,6 +3,12 @@
   <head>
     <meta charset="utf-8" />
     <link rel="icon" href="%PUBLIC_URL%/favicon.ico" />
+    <link
+  rel="stylesheet"
+  href="https://maxcdn.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.min.css"
+  integrity="sha384-ggOyR0iXCbMQv3Xipma34MD+dH/1fQ784/j6cY/iJTQUOhcWr7x9JvoRxT2MZw1T"
+  crossorigin="anonymous"
```
Comment (teodorvoicencu) [inline frontend/public/index.html:10]: Vlad, this bootstrap https://react-bootstrap.github.io/

Comment (konstrike) [reply frontend/public/index.html:10]: L-am pus pe ala, dar fara link nu-l vede

Comment (teodorvoicencu) [reply frontend/public/index.html:10]: Unde l-ai pus ca nu vad nimic instalat in `package.json` ? Trebuie sa te duci cu CLI in folderul cu frontend si sa dai `yarn install react-bootstrap bootstrap`, si dupa pui dependentele pt stil ...

File: frontend/src/Layout/TopNavBar.jsx [hunk 2]
```diff
@@ -0,0 +1,21 @@
+import React from 'react'
+
+import {Navbar, Nav} from 'react-bootstrap';
+
+
+function TopNavBar(){
```
Comment (teodorvoicencu) [inline frontend/src/Layout/TopNavBar.jsx:6]: `const TopNavBar = () : React.Node => (<> .... </>)` please do this for all the components

File: frontend/src/Layout/TopNavBar.jsx [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+    return (
+        <>
```
Comment (teodorvoicencu) [inline frontend/src/Layout/TopNavBar.jsx:8]: This fragment is useless

File: frontend/src/app/AppRouting.jsx [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+    return (
+        <>
+            <Route exact path="/" component={Home}></Route>
+            <Route exact path="/About" component={About}></Route>
+        </>
+    )
+    
+}
+
+export default AppRouting;
```
Comment (teodorvoicencu) [inline frontend/src/app/AppRouting.jsx:16]: please run `yarn fix`

File: frontend/src/app/AppRouting.jsx [hunk 5]
```diff
@@ -0,0 +1,16 @@
+import React from "react";
+import {Route } from "react-router-dom";
+import Home from './Pages/Home'
+import About from './Pages/About'
+
+function AppRouting(){
```
Comment (teodorvoicencu) [inline frontend/src/app/AppRouting.jsx:6]: There will be a lot of routes and this file will get bloated. We should use a json object with routing configurations and map all of these to Routes as it is done [here](https://alligator.io/react/react-router-map-to-routes/). Also please see [this](https://itnext.io/async-react-using-react-router-suspense-a86ade1176dc). We will fetch data at the entry of each route so the components should be async.

File: frontend/src/app/app.jsx [hunk 6]
```diff
@@ -1,32 +1,20 @@
...
-                    Learn React
-                </a>
-            </header>
-        </div>
+        <Router>
+            <TopNavBar />
+
+            <AppRouting />
+
+        </Router>
```
Comment (teodorvoicencu) [inline frontend/src/app/app.jsx:17]: Please run yarn fix in the console to format the code

**Pull request closed without merging**